src/java/org/apache/cassandra/index/sai/disk/v1/SSTableIndexWriter.java

```diff
@@ -122,7 +126,7 @@ public void complete(Stopwatch stopwatch) throws IOException
         long elapsed;
 
         boolean emptySegment = currentBuilder == null || currentBuilder.isEmpty();
-        logger.debug(indexContext.logMessage("Completing index flush with {}buffered data..."), emptySegment ? "no " : "");
+        logger.debug("Completing index flush with {}buffered data...", emptySegment ? "no " : "");
```

Comment:
It's logged when compaction starts, I don't think it's good practice to include it redundantly (it's redundant since there's only a single thread involved), especially when it's not done consistently (~nowhere but this file, so all the other log messages associated with the compaction don't have it)

Reply:
#collaborating

Reply:
Since it's the same thread, that works for me.